### PR TITLE
Fixed feature flag behavior

### PR DESCRIPTION
**Pull request by adeliaferreira**
# Description

The feature flag behavior was inverter, now is working as expected.   

Fixes # (issue)

## Type of change

What is it?

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I run `npm run lint:js:fix` to check that my code is properly formatted

### CODE REVIEW

File: src/Routes/Devices/DeviceTable.js [hunk 1]
```diff
@@ -307,7 +307,7 @@ const DeviceTable = ({
     if (isLoading) return actions;
     if (!rowData?.rowInfo?.id) return actions;
 
-    if (handleAddDevicesToGroup && groupActionsEnabled) {
+    if (handleAddDevicesToGroup && !groupActionsEnabled) {
```
Comment (ldjebran) [inline src/Routes/Devices/DeviceTable.js:310]: maybe rename the variable groupActionsEnabled to hideGroupsActions

Comment (adeliaferreira) [reply src/Routes/Devices/DeviceTable.js]: Renamed, thanks for the suggestion

File: src/Routes/Devices/Inventory.js [hunk 2]
```diff
@@ -265,7 +265,7 @@ const Inventory = ({
           selectedItems={setCheckedDeviceIds}
           selectedItemsUpdateable={canBeUpdated()}
           kebabItems={
-            groupActionsEnabled
+            !groupActionsEnabled
```
Comment (ldjebran) [inline src/Routes/Devices/Inventory.js:268]: I liked the latest changes too much , can you rename this variable to  hideGroupsActions and do the same changes as in DeviceTable 
and use the const FEATURE_HIDDEN_GROUP_ACTIONS

File: src/constants/features.js [hunk 3]
```diff
@@ -1,2 +1,5 @@
 export const FEATURE_PARITY_INVENTORY_GROUPS =
   'edgeParity.inventory-groups-enabled';
+
+export const FEATURE_HIDDEN_GROUP_ACTIONS =
```
Comment (ldjebran) [inline src/constants/features.js:4]:
```suggestion
export const FEATURE_HIDE_GROUPS_ACTIONS =
```

**Review by ldjebran — COMMENTED** [review]
looks awesome , put some comment to align the changes.

File: src/constants/features.js (hunk 3 above)
Comment (adeliaferreira) [reply src/constants/features.js]: renamed

File: src/Routes/Devices/Inventory.js (hunk 2 above)
Comment (adeliaferreira) [reply src/Routes/Devices/Inventory.js]: changes applied, thanks

**Review by ldjebran — APPROVED** [review]
lgtm

**Review by de1987 — APPROVED** [review]

**Pull request merged**